src/proto/mgmt/pool.proto

```diff
@@ -229,13 +229,14 @@ message PoolQueryResp {
 	PoolRebuildStatus rebuild = 7; // pool rebuild status
 	repeated StorageUsageStats tier_stats = 8; // storage tiers usage stats
 	uint32 version = 10; // latest pool map version
-	uint32 leader = 11; // current raft leader
+	uint32                     leader             = 11; // current raft leader (2.4)
 	string enabled_ranks = 12; // optional set of ranks enabled
 	string disabled_ranks = 13; // optional set of ranks disabled
 	uint32 total_engines = 14; // total engines in pool
 	uint32 pool_layout_ver = 15; // current pool global version
 	uint32 upgrade_layout_ver = 16; // latest pool global version to upgrade
 	PoolServiceState state = 17; // pool state
+	uint32                     svc_ldr            = 18; // current raft leader (2.6+)
```

Comment:
Weird indentation on the new fields.

Reply:
Ugh, thanks clang-format. I think maybe something got changed because it didn't use to try to mess with the .proto or generated files.

Reply:
Weird, on my system it doesn't seem to do anything to the .proto files.